packages/remix-dev/package.json

```diff
@@ -36,7 +36,7 @@
     "chokidar": "^3.5.1",
     "dotenv": "^16.0.0",
     "esbuild": "0.17.6",
-    "esbuild-plugins-node-modules-polyfill": "^1.3.0",
+    "esbuild-plugins-node-modules-polyfill": "^1.4.0",
```

Comment:
This introduces the new `fallback: "empty"` option: https://github.com/imranbarbhuiya/esbuild-plugins-node-modules-polyfill/pull/146